src/pages/quotes/common/components/QuoteStatus.tsx

```diff
@@ -39,8 +43,5 @@ export function QuoteStatus(props: Props) {
       return <Badge variant="dark-blue">{t('approved')}</Badge>;
     default:
       return <Badge variant="light-blue">{t('error')}</Badge>;
-      break;
   }
-
-  return <></>;
```

Comment:
Same case for the rest, make sure to use enums.

https://github.com/invoiceninja/ui/blob/main/src/common/enums/quote-status.ts#L13

Reply:
done